config/manager/controller-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: numaplane-controller-config
+data:
+  config.yaml: |
+    syncTimeIntervalMs: 30000
```

Comment:
not using `syncTimeIntervalMs`; are we using `includedResources`?

Reply:
should logLevel be in here?

Reply:
Log an issue to track it.

Reply:
https://github.com/numaproj/numaplane/issues/24